pkg/ocicni/ocicni.go

```diff
@@ -335,9 +342,15 @@ func (plugin *cniNetworkPlugin) syncNetworkConfig() error {
 
 	plugin.Lock()
 	defer plugin.Unlock()
-	if plugin.defaultNetName == "" {
+
+	// update default CNI network is defaultNetName is not specified during initialization
+	if plugin.isDefaultNetNameSpecified {
+		logrus.Infof("default CNI network %s is specified during initialization and can not be changed", plugin.defaultNetName)
```

Comment:
Unnecessary to print this prompt every time?

Reply:
I think it's a small probability event ~ so, forget it :)